docs/tutorials/getting_started/tournament_results.rst

```diff
@@ -0,0 +1,246 @@
+.. _tournament-results:
+
+Accessing tournament results
+============================
+
+This tutorial will show you how to access the various results of a tournament:
+
+- Wins: the number of matches won by each player
+- Match lengths: the number of turns of each match player by each player
```

Comment:
I don't get what this means myself, so I'm not sure a beginner would. I'll have to look at the code to suggest some alternative.